modules/oadp-restoring-applications.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing-up-and-restoring-applications.adoc
+
+[id="oadp-restoring-applications_{context}"]
+= Restoring applications
+
+You restore applications by creating a `Restore` custom resource (CR).
+
+.Prerequisites
+
+* Velero `Backup` CR created.
```

Comment:
It would be good to add the prerequisites of both the OADP operator and DPA CR here.

Also, if there are backups already in the BSL, then velero will sync backups to this cluster. Maybe just that a velero backup resource must exist?